src/menu/StudentMenu.js

```diff
@@ -1,20 +1,36 @@
 import { MultiLevelMenu, MenuItemCategory } from '@react-admin/ra-navigation'
 import { useNotify } from 'react-admin'
+import Button from '@mui/material/Button'
+import Box from '@mui/material/Box'
 
+import LogoutIcon from '@mui/icons-material/Logout'
 import { Receipt, AttachMoney, AccountCircle } from '@mui/icons-material'
 
 import authProvider from '../providers/authProvider'
 
 export const StudentMenu = () => {
+  const logout = () => {
+    authProvider.logout()
+  }
+
   const notify = useNotify()
   const notifyNotImplemented = () => notify('En cours de développement. Ce qui présage quelques exercices pour vous 😉', { type: 'warning' })
   const whoamiId = authProvider.getCachedWhoami().id
   return (
-    <MultiLevelMenu variant='categories'>
-      <MenuItemCategory to='/profile' name='profile' label='Mon profil' icon={<AccountCircle />} />
-      <MenuItemCategory to={whoamiId ? `/students/${authProvider.getCachedWhoami().id}/fees` : '/'} name='fees' label='Frais' icon={<AttachMoney />} />
-      <MenuItemCategory to='/' name='student-grades' label='Notes' icon={<Receipt />} onClick={notifyNotImplemented} />
-    </MultiLevelMenu>
+    <Box>
+      <Box>
+        <MultiLevelMenu variant='categories' sx={{ width: '200px', backgroundColor: '#2c393a', color: 'white', pb: '480px' }}>
+          <MenuItemCategory to='/profile' name='profile' label='Mon profil' icon={<AccountCircle />} />
+          <MenuItemCategory to={whoamiId ? `/students/${authProvider.getCachedWhoami().id}/fees` : '/'} name='fees' label='Frais' icon={<AttachMoney />} />
```

Comment:
C6. use the retrieved whoamiId instead of re-getting cachedWhoami.id

whoamiId ? \`students/${whoamiId}/feeds\` : ''